tgbotapi.api/src/commonMain/kotlin/dev/inmo/tgbotapi/extensions/api/gifts/GiftPremiumSubscription.kt

```diff
@@ -0,0 +1,30 @@
+package dev.inmo.tgbotapi.extensions.api.gifts
+
+import dev.inmo.tgbotapi.bot.TelegramBot
+import dev.inmo.tgbotapi.requests.gifts.GiftPremiumSubscription
+import dev.inmo.tgbotapi.types.UserId
+import dev.inmo.tgbotapi.types.message.ParseMode
+import dev.inmo.tgbotapi.types.message.textsources.TextSourcesList
+
+public suspend fun TelegramBot.giftPremiumSubscription(
+    userId: UserId,
+    monthCount: Int,
+    starCount: Int,
+    text: String,
+    parseMode: ParseMode?
+): Boolean = execute(
+    GiftPremiumSubscription(
+        userId, monthCount, starCount, text, parseMode
+    )
+)
+
+public suspend fun TelegramBot.giftPremiumSubscription(
+    userId: UserId,
+    monthCount: Int,
+    starCount: Int,
+    textSources: TextSourcesList? = null,
```

Comment:
I am not sure that it can be null here

Reply:
![image](https://github.com/user-attachments/assets/57996adb-1deb-425c-8e3d-bfc8f7655aa7)

It can. Text is completely optional.